### PR TITLE
Chore/sc 23819/update nice challenge collection page

### PR DESCRIPTION
**Pull request by pmanal1**
[23819](https://app.shortcut.com/clarkcan/story/23819/update-nice-challenge-collection-page)

## Demo

https://github.com/Cyber4All/clark-client/assets/93054689/2b8fd264-83e4-4dcd-b9c5-1610cc7a8782

### CODE REVIEW

**Review by bereketsega — APPROVED** [review]

File: src/app/collection/shared/included/feature/feature.component.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
              <h2 class="featured-learning-title">
                  FEATURED LEARNING OBJECTS
              </h2>
-             <p>
+             <p *ngIf="collectionName !== 'nice' else niceDescription">
```
Comment (pzalep1) [inline src/app/collection/shared/included/feature/feature.component.html:8]: Why was this added?

**Review by mfranz1 — APPROVED** [review]
Just Paige's question, looks good though.

File: src/app/collection/shared/included/feature/feature.component.html (hunk 1 above)
Comment (pmanal1) [reply src/app/collection/shared/included/feature/feature.component.html:8]: This component was in shared, so I thought it was being used in other places, so I made it so that the changes in this component are just to the nice collection. I know that it's actually only being used for the nice collection, would it be better to remove the conditional?

**Review by pzalep1 — APPROVED** [review]
